examples/token-stats/README.md

```diff
@@ -0,0 +1,48 @@
+# Compound Token Stats (APR Proof)
+
+**An example that calls multiple view function on the Compound USDC contract to compute the APR.**
+
+> ***WARNING***: This software is still experimental, we do not recommend it for production use.
+
+## Prerequisites
+
+To get started, you need to have Rust installed. If you haven't done so, follow the instructions [here][install-rust].
+
+Next, install the `cargo risczero` tool. We'll use `cargo binstall` to facilitate this. Detailed instructions can be found at [cargo-binstall].
+
+```bash
+cargo install cargo-binstall
+cargo binstall cargo-risczero
```

Comment:
Some trickiness with this is that, on main, they will end up using two different versions of the zkVM. One is specified in the Cargo.toml here as the `risc0-zkvm` dependency and will be `main`, and the other is the latest release installed by `cargo binstall`. It's unclear whether we are guaranteeing this will work or not, post 1.0.

At the very least, the result right now is
```
called `Result::unwrap()` on an `Err` value: incompatible client version: 1.1.0-alpha.1, server version: 1.0.1
```

This issue is not unique to this example, and we are trying to deal with this on `risc0` as well.

Reply:
Might be better to just link to the install page on the docs so that if we make changes to the installation (like moving to `rzup`), we don't need to update all the different descriptions of installation.